scripts/generator/graphql_generator/csharp/Cart.cs.erb

```diff
@@ -26,9 +26,25 @@ namespace <%= namespace %> {
             }
         }
 
-        private CartLineItems _LineItems;
+        private bool IsSaved {
+            get {
+                return IsCreated && 
+                    LineItems.New().Count == 0 &&
+                    LineItems.Modified().Count == 0 &&
+                    LineItems.Deleted().Count == 0;
+            }
+        }
 
+        private bool IsCreated {
+            get {
+                return CurrentCheckout != null;
+            }
+        }
+
+        private const float POLL_DELAY = 0.5f;
+        private CartLineItems _LineItems;
```

Comment:
Is the `_` a convention thing around `private`? If so, then should we do this on line 47 as well?